storage/Devices/LvmPvImpl.cc

```diff
@@ -46,6 +46,8 @@ namespace storage
 
     const char* DeviceTraits<LvmPv>::classname = "LvmPv";
 
+    const unsigned long long LvmPv::Impl::default_pe_start;
```

Comment:
shouldn't be const initialized here?

Reply:
or better why it is needed also here, when it is already initialized in header? I still do not get why this change is needed.

Reply:
In C/C++ there is the declaration and the definition of objects. The header provides the declaration (maybe with initialization) but not the definition. Normally both declaration and definition are needed.

What is special here is that with optimization the compiler generates code that does not need the address of the variable (thus it does not need the definition). Since I normally compile with optimization I did not notice the missing definition right away.